lsp-mode.el

```diff
@@ -4480,8 +4480,7 @@ and the position respectively."
          (inhibit-field-text-motion t))
     (save-excursion
       (goto-char point)
-      (buffer-substring-no-properties (line-beginning-position)
```

Comment:
The buffer substring no properties is smaller and potential faster

Reply:
It might be, but it removes syntax highighting from xref.

Reply:
What I mean is this: `ivy-xref' uses that string, with its properties, to show references (that string becomes `xref-item-summary`). If the properties are removed, the text becomes gray, which is arguably worse than a pretty, syntax-highlighted usage line.

Reply:
I see. nice

Reply:
I think you should rebase and create separate branch for each PR. For now, it will mix thing too much if you make PR right from your `master` branch.